.github/workflows/main.yml

```diff
@@ -0,0 +1,559 @@
+#
+# Copyright (c) 2014-present, The osquery authors
+#
+# This source code is licensed as defined by the LICENSE file found in the
+# root directory of this source tree.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+#
+
+# Due to a limitation in how GitHub Actions works, we can't reference
+# jobs in another file inside the `needs` statement.
+#
+# This configuration file takes care of the Windows, macOS and Linux
+# builds on the x86 platform.
+name: Main
+
+on:
+  push:
+    branches:
+      - '*'
+
+    tags:
+      - 'v*'
+
+  pull_request:
+    branches:
+      - '*'
+
+# If the initial code sanity checks are passing, then one job
+# per [`platform` * `build_type`] will start, building osquery
+# and generating packages that are later attached to the commit
+# (or PR) as build artifacts.
+#
+# The checkout@v2 action removes access to the `.git` folder, so
+# we have to use v1 instead
+jobs:
+
+  # This job performs basic source code check, looking for formatting
+  # issues and missing copyright headers
+  check_source_code:
+    runs-on: ubuntu-18.04
+
+    container:
+      image: trailofbits/osquery:ubuntu-18.04-toolchain-v9
```

Comment:
(not blocking) where's the source for this?

Reply:
This is the same image we use in Azure Pipelines, it comes from here: https://github.com/osquery/osquery/blob/master/tools/ci/osquery-ubuntu18.04-toolchain.dockerfile